string mode_name
---
bool ok
uint8 error